apps/src/sharedComponents/footer/copyright-dialog.module.scss

```diff
@@ -0,0 +1,23 @@
+@import 'font.scss';
+@import 'color.scss';
+@import "@cdo/apps/componentLibrary/typography/typography.module";
+
+.copyright {
+    hr {
+        border-color: $lighter_gray;
+    }
+    p {
+        @include body-three;
```

Comment:
I tried wrapping the typography `BodyThreeText`  around Safe markdown, but `BodyThreeText` provides a `p` element, and SafeMarkdown wraps everything around a div element.
```
console.js:288 Warning: validateDOMNesting(...): <p> cannot appear as a descendant of <p>.
```
As a workaround, I imported the `body-three` style from typography and applied it to every `p` inside `.copyright`